pythonforandroid/recipes/scipy/__init__.py

```diff
@@ -87,5 +88,9 @@ def get_recipe_env(self, arch):
         env['LDFLAGS'] += f' -l{self.stl_lib_name}'
         return env
 
+    def build_arch(self, arch):
+        self.install_hostpython_prerequisites()
+        super().build_arch(arch)
```

Comment:
Minor (but let's fix it separately): I see we're adding `self.install_hostpython_prerequisites()` in a few places, we can likely avoid it, and just try the installation by default.